components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -193,22 +188,49 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
                 </div>
             );
 
-            newDropBox = (
-                <div
-                    ref={this.newDropBoxRef}
-                    className={classNames('SidebarCategory_newDropBox', {
-                        collapsed: isCollapsed || (draggingState.type === DraggingStateTypes.CATEGORY && draggingState.id === category.id),
-                    })}
-                    onTransitionEnd={this.removeAnimation}
-                >
-                    <i className='icon-hand-right'/>
-                    <span className='SidebarCategory_newDropBox-label'>
-                        <FormattedMessage
-                            id='sidebar_left.sidebar_category.newDropBoxLabel'
-                            defaultMessage='Drag channels here...'
-                        />
-                    </span>
-                </div>
+            newDropBox = (isDraggingOver: boolean) => (
+                <React.Fragment>
+                    <Draggable
+                        draggableId={'FAKE_CHANNEL'}
```

Comment:
This is a hack done to make sure the drop target for the first channel in a new category floats below the category header, since we need the header to be part of the droppable area, and `react-beautiful-dnd` will automatically float the first channel dragged in into the header if we don't have something else in there to provide space.

I'll add a comment since it's definitely unclear.

Reply:
Maybe a different name might be helpful. Maybe it's because I'm old, but we have somewhere else that channels are "fake", so I might be confusing it with that.

Reply:
Sure, I can change the name.